admin/context_test.go

```diff
@@ -0,0 +1,58 @@
+package admin_test
```

Comment:
because it takes time to switch package for these old tests. I'm working on CI now. will migrate them later.